src/components/search-lightbox/script.js

```diff
@@ -22,9 +27,9 @@ export default {
   watch: {
     dialog(value) {
       if (!value) {
-        this.results = [];
-        this.term = null;
         this.loading = false;
+        this.results = [];
+        this.$refs['search-input'].reset();
```

Comment:
Why use array notation instead of object notation?
I think we could use a camelcase notation to simplify the code and improve readability.

Reply:
Fixed